components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/LoginResource.java

```diff
@@ -86,10 +87,17 @@ public boolean invoke(MessageContext messageContext, org.apache.axis2.context.Me
         String username = messageContext.getProperty(USERNAME_PROPERTY).toString();
         JWTTokenInfoDTO newToken = new JWTTokenInfoDTO(username);
         newToken.setToken(randomUUIDString);
-        if (isAdmin(username)) {
-            newToken.setScope(AuthConstants.JWT_TOKEN_ADMIN_SCOPE);
-        } else {
-            newToken.setScope(AuthConstants.JWT_TOKEN_DEFAULT_SCOPE);
+        try {
+            if (MicroIntegratorSecurityUtils.isAdmin(username)) {
+                newToken.setScope(AuthConstants.JWT_TOKEN_ADMIN_SCOPE);
+            } else {
+                newToken.setScope(AuthConstants.JWT_TOKEN_DEFAULT_SCOPE);
+            }
+        } catch (UserStoreException e) {
+            final String errorMessage = "Error occurred authenticating user. ";
```

Comment:
Removed an additional space :) 

```suggestion
            final String errorMessage = "Error occurred authenticating user.";
```